linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_server.js

```diff
@@ -0,0 +1,28 @@
+var RouterServer = (function() {
+  // Skeletal implementation to demonstrate functionality of RouterController
+
+  function initializeServerContainers(servers, $serverEl) {
```

Comment:
Cool, yeah, totally fine by me. Just wanted to make sure I understood what was up.